src/Microsoft.AspNetCore.Mvc.Abstractions/Abstractions/ActionDescriptor.cs

```diff
@@ -53,6 +53,6 @@ public ActionDescriptor()
         /// <summary>
         /// Stores arbitrary metadata properties associated with the <see cref="ActionDescriptor"/>.
         /// </summary>
-        public IDictionary<object, object> Properties { get; }
```

Comment:
Made these get/set for unit testing. It's consistent with everything else in this class